platform/src/main/java/org/stellar/anchor/platform/data/JdbcSep24Transaction.java

```diff
@@ -60,12 +63,19 @@ public class JdbcSep24Transaction implements Sep24Transaction, SepTransaction {
   @SerializedName("completed_at")
   Instant completedAt;
 
+  @SerializedName("updated_at")
+  Instant updatedAt;
```

Comment:
Shouldn't you update Sep24Transaction interface as well?

Reply:
`updated_at` is not part of the SEP-24. It is for the platform API. So, we do not include it in the core interface.

Reply:
This leaves the possibilities for partners who would like to implement their own integration (database, etc) without the platform module.

Reply:
You are right